# Detailed information about the amount of satellites
# custom sat_block_t
	uint16 TotalSats  # Total satellites in this block
	uint8 NumGps  # Number of GPS satellites
	uint8 NumGlo  # Number of GLONASS satellites
	uint8 NumGal  # Number of Galileo satellites
	uint8 NumBds  # Number of Beidou satellites
	uint8 NumSbas  # Number of SBAS satellites
	uint8 NumQzss  # Number of QZSS satellites
	uint8 NumIrnss  # Number of IRNSS satellites
	uint8 NumLband  # Number of L-Band satellites
